src/app/core/services/storage/storage.service.ts

```diff
@@ -9,8 +9,11 @@ export class StorageService {
   global: { [key: string]: any } = {}
 
   constructor(private storage: Storage, private logger: LogService) {
-    this.prepare()
-      .then(() => this.logger.log('Global configuration', this.global))
+    this.prepare().then(() => {
+      // NOTE: Keeping console.log for web inspection
+      console.log(this.global)
```

Comment:
Can this be fixed in `LogService` instead? It now checks whether the platform is `desktop`, in that case it logs the object directly. Is the platform something different in your case?

Reply:
Yeah, it's `core` or `mobileweb` during `ionic serve` in my case. But also for inspecting (`chrome://inspect`) the android phone itself, the unformatted object version is easier to read but the platform would be `android` in this case.

Reply:
I'll make a separate fix then. Could you please remove this additional call?

Reply:
Okay sure

Reply:
fixed in #611